app/config/index.js

```diff
@@ -100,11 +100,11 @@ value.isTest = value.env === 'test'
 value.isProd = value.env === 'production'
 
 value.catboxOptions = {
-  host: value.redisHost,
-  port: value.redisPort,
-  password: value.redisPassword,
-  tls: value.isProd ? {} : undefined,
-  partition: value.redisPartition
+  host: value.cache.options.host,
+  port: value.cache.options.port,
+  password: value.cache.options.password,
+  tls: value.cache.options.tls,
+  partition: value.cache.options.partition
```

Comment:
I think we should change the partition name values so portal and enforcement have different settings, otherwise it will be difficult to prove if one or both are working i.e. currently both are set to 'dangerous-dog-act-portal'